sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -745,7 +756,12 @@ async def get_throughput(self, **kwargs: Any) -> ThroughputProperties:
             response_hook(self.client_connection.last_response_headers, throughput_properties)
 
         return ThroughputProperties(offer_throughput=throughput_properties[0]["content"]["offerThroughput"],
-                                    properties=throughput_properties[0])
+                                    properties=throughput_properties[0],
+                                    auto_scale_max_throughput=throughput_properties[0]['content']
+                                    ['offerAutopilotSettings']['maxThroughput'],
+                                    auto_scale_increment_percent=throughput_properties[0]['content']
+                                    ['offerAutopilotSettings']['autoUpgradePolicy']['throughputPolicy'][
+                                        'incrementPercent'])
 
```

Comment:
still unsure about how this will work for containers that don't have autoscale settings